### PR TITLE
add user posts list

### PR DESCRIPTION
**Pull request by shivansh-pandeyy**
Screenshot from 2022-10-18 20-17-48

### CODE REVIEW

File: src/components/CardComponent/CardComponent.tsx [hunk 1]
```diff
@@ -7,26 +7,40 @@ import {
...
 interface CardComponentProps {
   name?: string;
-  image: string;
+  image?: string;
   showMenu: boolean;
   menuWithAction?: MenuAction[];
+  onClick?: () => void;
+  children?: JSX.Element;
+  sx?: SxProps;
+  title?: String;
```
Comment (anand-velotio) [inline src/components/CardComponent/CardComponent.tsx]:
```suggestion
  title?: string;
```

File: src/components/Header/Header.scss [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  border: 2px solid lightcoral;
+  border-radius: 4px;
+  padding: 0.5rem 1rem;
+  color: lightcoral;
+}
+
+.active {
+  background-color: lightsalmon;
+  color: white;
+}
```
Comment (anand-velotio) [inline src/components/Header/Header.scss]:
```suggestion
.tab {
  border: 2px solid lightcoral;
  border-radius: 4px;
  padding: 0.5rem 1rem;
  color: lightcoral;
    &.active {
      background-color: lightsalmon;
      color: white;
    }
}


```

Comment (anand-velotio) [reply src/components/Header/Header.scss]: Since this is scss, it will be written like this.

File: src/views/Posts/Posts.tsx [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+          {
+            label: 'Posts',
+            to: `/users/${params.id}/posts`,
+            active: true,
+          },
+          {
+            label: 'Todo',
+            to: `/users/${params.id}/todos`,
+          },
+        ]}
```
Comment (anand-velotio) [inline src/views/Posts/Posts.tsx]: Move the list outside the component. We don't need to re-render it multiple times.

**Review by anand-velotio — CHANGES_REQUESTED** [review]

File: src/components/CardComponent/CardComponent.tsx (hunk 1 above)
Comment (shivansh-pandeyy) [reply src/components/CardComponent/CardComponent.tsx]: done

File: src/components/Header/Header.scss (hunk 2 above)
Comment (shivansh-pandeyy) [reply src/components/Header/Header.scss]: done

File: src/views/Posts/Posts.tsx (hunk 3 above)
Comment (shivansh-pandeyy) [reply src/views/Posts/Posts.tsx]: done

**Review by anand-velotio — APPROVED** [review]

**Pull request merged**